.github/workflows/release-clarity-vscode.yaml

```diff
@@ -0,0 +1,108 @@
+name: Build Release for Clarity VSCode
+on:
+  pull_request:
+  push:
+    tags:
+     - 'v*'
```

Comment:
Could we use a prefix the git tag with a namespace, so something like `vscode-v*`?

Reply:
That would be super useful indeed.
Not sure if it's super common to handle multiple release / version flow with a tag like that but this would keep a nice developer experience (thanks to the single repo) and add automatic CD

Reply:
cc @aravindgee, thoughts?